components/collections/codepointtrie_builder/Cargo.toml

```
@@ -27,14 +27,28 @@ include = [
 [package.metadata.workspaces]
 independent = true
 
+[features]
+# Use the wasm builder
+default = ["wasm"]
+wasm = ["wasmer", "wasmer-wasi"]
+# Use the ICU4C builder
+# needs the ICU4C_LIB_PATH variable set and pointing to an ICU4C lib folder
+# containing dylibs. If you want to use staticlibs, set ICU4C_LINK_STATICALLY
+# Will be silently disabled if the wasm feature is enabled
+icu4c = ["zerovec"]
+# Use icu4c with unsuffixed APIs (u_foobar instead of u_foobar_72)
+icu4c_unsuffixed = ["icu4c"]
```

The ICU4C flag is called U_DISABLE_RENAMING so let's use the same terminology here

```suggestion
icu4c_disable_renaming = ["icu4c"]
```